src/resources/views/crud/inc/filters_navbar.blade.php

```diff
@@ -62,16 +62,19 @@ function updatePageUrl(filterName, filterValue, currentUrl = null) {
     }
 
     if(typeof updateDatatablesOnFilterChange !== 'function') {
```

Comment:
Todo: after the review, we should probably move this JS logic from the FILTERS blade file... to the DATATABLES blade file. Right? I mean... filters are a general-purpose feature... that is supported by whatever operation wants to support them.

DataTables - as a component and as an operation... chooses to support the filters. So the logic that says "_when filter does X datatables does Y_" should probably live in the DATATABLE, not in the FILTERS.

Reply:
But right now it's ok here, just to keep the diff minimal. We can create a different PR that moves code around. Like a cleanup PR.